impexp-kml-collada-plugin/src/main/java/org/citydb/modules/kml/database/KmlExporterManager.java

```diff
@@ -655,15 +655,32 @@ private void convertColladaToglTF(ColladaBundle colladaBundle, File buildingDire
 		String collada2gltfPath = config.getProject().getKmlExporter().getPathOfGltfConverter();
 		File collada2gltfFile = new File(ClientConstants.IMPEXP_HOME.resolve(collada2gltfPath).toString());
 		if (collada2gltfFile.exists()) {
-			ProcessBuilder pb = new ProcessBuilder(collada2gltfFile.getAbsolutePath(), "-i", colladaBundle.getGmlId() + ".dae", "-o", gltfModelFile.getAbsolutePath(), "-v", exportGltfV1 ? "1.0" : "2.0");
+			List<String> commands = new ArrayList<>();
+			commands.add(collada2gltfFile.getAbsolutePath());
+			commands.add("-i");
+			commands.add(colladaModelFile.getAbsolutePath());
+			commands.add("-o");
+			commands.add(gltfModelFile.getAbsolutePath());
+			commands.add("-v");
+			commands.add(exportGltfV1 ? "1.0" : "2.0");
+			if (!config.getProject().getKmlExporter().isEmbedTexturesInGltfFiles()) {
+				commands.add("-t");
+			}
+			if (config.getProject().getKmlExporter().isExportGltfBinary()) {
+				commands.add("-b");
+			}
+			if (config.getProject().getKmlExporter().isEnableGltfDracoCompression()) {
+				commands.add("-d");
+			}
+			ProcessBuilder pb = new ProcessBuilder(commands);
 			pb.directory(buildingDirectory);
 			try {
 				Process process = pb.start();
 				process.waitFor();
 			} catch (IOException | InterruptedException e) {
-				log.debug("Unexpected errors occurred while converting collada to glTF for city object '" + colladaBundle.getGmlId() + "' with output path: '" + gltfModelFile.getAbsolutePath() + "'");
+				log.debug("Unexpected errors occurred while converting collada to glTF for city object '" + colladaBundle.getGmlId() + "' with output path: '" + gltfModelFile.getAbsolutePath() + "'" + "\n" + e.getMessage());
 			} finally {
-				if (config.getProject().getKmlExporter().isNotCreateColladaFiles() && gltfModelFile.exists()) {
+				if (config.getProject().getKmlExporter().isNotCreateColladaFiles() && (gltfModelFile.exists() || (new File(gltfModelFile.getAbsolutePath().replace(".gltf", ".glb"))).exists())) {
```

Comment:
The check for existence of the glTF file was already there before this PR. Before this PR, it was understood that if the option to not create COLLADA files is selected and there are already glTF files, then the program can proceed to deleted the COLLADA files. Now using the same logic, we can also check if either glTF or glb files exist, since if the option to export glb is selected, only glb (instead of glTF) files will be generated.